MdsSupportingClasses/Collivery.php

```diff
@@ -315,6 +316,44 @@ public function searchTowns($name)
         }
     }
 
+
+    /**
+     * Allows you to search for town and suburb names starting with the given string.
+     * The minimum string length to search is three characters.
+     * Returns a list of suburbs and the towns the suburbs belong to with their ID's for creating new addresses.
+     *
+     * @param string $search_text Start of town/suburb name
+     *
+     * @return array List of towns and their suburbs with their ID's
+     * @throws Exception
+     */
+    public function searchTownSuburbs($search_text)
+    {
+        if (strlen($search_text) < 3) {
+            return $this->getSuburbs();
```

Comment:
This will error out being that `getSuburbs()` has a required parameter of `$town_id`